components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/dao/impl/ApplicationDAOImpl.java

```diff
@@ -416,9 +390,51 @@ public void addApplicationKeys(String appId, String keyType, OAuthApplicationInf
         }
     }
 
-    /**
-     * @see ApplicationDAO#getLastUpdatedTimeOfApplication(String)
-     */
```

Comment:
Are we not using @see for method comments referring? Anyway better to add method comments.